### DISCUSSION TITLE
Remove padding from Menu

### DISCUSSION THREAD

**Discussion by orenmizr** [chakra-ui/chakra-ui#4476] [Q&A] [answered]
the default menu comes with **top and bottom padding** - https://chakra-ui.com/docs/overlay/menu
it is visible via css devtools.

i have been trying to remove unsuccessfully via props and theme styles without luck. (p={0} etc)
does anyone know how ?

**Comment by payapula** [comment, marked as answer]
@orenmizr You have to give `p={0}` to the `MenuList` component. Here is the [example](https://codesandbox.io/s/damp-firefly-6z7d4?file=/src/index.js:259-618).

```jsx
<Menu>
          <MenuButton as={Button}>Actions</MenuButton>
     ---->     <MenuList p={0}>
            <MenuItem>Download</MenuItem>
            <MenuItem>Create a Copy</MenuItem>
            <MenuItem>Mark as Draft</MenuItem>
            <MenuItem>Delete</MenuItem>
            <MenuItem>Attend a Workshop</MenuItem>
          </MenuList>
  </Menu>
```

**Reply by orenmizr** [reply]
my bad. i put it on menu rather than menulist. thanks so much for helping out 🙏
